src/plugins/intel_gpu/tests/unit/test_cases/permute_gpu_test.cpp

```diff
@@ -1677,7 +1757,7 @@ void TiledPermuteTest::run_test(const std::vector<cldnn::tensor::value_type>& si
     }
 
     auto input = engine.allocate_memory({Data_Type, format, tensor});
-    set_random_values<type>(input);
+    set_values<type>(input);
```

Comment:
why did you change this ? It is setting input only filled with "1"s...which is not good for test.

Reply:
Reverted.